src/features/blocks/BlocksCategory.tsx

```diff
@@ -32,23 +23,22 @@ function BlocksCategory(props: ToolboxCategory) {
         blocksContainer.style.maxHeight = maxHeight + "px"
       }
     })()
-  }, [props.toggled, props.blocks])
+  }, [open, props.openDesc])
```

Comment:
Nicely done! If it doesn't need to be in the Redux state, using a local component state is better :+1:

Reply:
Thank you & roger that!